easybuild/easyblocks/l/lapack.py

```diff
@@ -100,11 +100,32 @@ def configure_step(self):
         else:
             copy_file(src, dest)
 
+        # control compiler commands being used
+        self.cfg.update('buildopts', 'CC="%s"' % os.getenv('CC'))
+        # older versions use FORTRAN, newer versions use FC for Fortran compiler command
+        if LooseVersion(self.version) >= LooseVersion('3.9.0'):
+            fc_var = 'FC'
+        else:
+            fc_var = 'FORTRAN'
+        self.cfg.update('buildopts', '%s="%s"' % (fc_var, os.getenv('FC')))
+
         # set optimization flags
+
+        self.cfg.update('buildopts', 'CFLAGS="%s -m64"' % os.getenv('CFLAGS'))
+
+        # Fortran compiler flags are controlled via OPTS in older version, via FFLAGS in newer versions
+        if LooseVersion(self.version) >= LooseVersion('3.9.0'):
+            fflags_var = 'FFLAGS'
+            noopt_var = 'FFLAGS_NOOPT'
+        else:
+            fflags_var = 'OPTS'
+            noopt_var = 'NOOPT'
+        self.cfg.update('buildopts', '%s="%s -m64"' % (fflags_var, os.getenv('FFLAGS')))
```

Comment:
Not sure if explicitly setting -m64 is a good thing here and below.

Reply:
I'm happy to drop this, but it's just copying what the LAPACK `Makefile` does.

Any specific reason you think this is not a good idea?

Reply:
There is no -m64 anywhere in the LAPACK files whatsoever.
-m64 is compiler specific, not all compilers has that flag.